README.md

```diff
@@ -7,7 +7,7 @@
 - **Field Name Prefix:** dlm
 - **Scope:** Item, Collection
 - **Extension [Maturity Classification](https://github.com/radiantearth/stac-spec/tree/master/extensions/README.md#extension-maturity):** Proposal
-- **Owner**: @sfoucher  @person2
+- **Owner**: @sfoucher @fmigneault @ymoisan
```

Comment:
Good for me, even though my work on this specific spec is rather limited.

Reply:
Good for me, thank you Francis